account_statement_import_sheet_file/models/account_statement_import_sheet_parser.py

```diff
@@ -179,7 +179,7 @@ def _parse_lines(self, mapping, data_file, currency_code):
             csv_or_xlsx = reader(StringIO(decoded_file), **csv_options)
         header = False
         if not mapping.no_header:
```

Comment:
**question**: shouldn't the `if mapping.offset_column` (line 191) be intended under `if not mapping.no_header` (line 181)?

Reply:
I think it's the right time to fix it! 

For the future:
We moved the `offset_column` logic under `if not mapping.no_header:` because otherwise it would try to index slice on a `False` value, which would raise an error.